### ISSUE TITLE
Final Target ES 6?

### ISSUE THREAD

**Issue by newtack** [fuse-box/fuse-box-rollup-example#1]
If the final target is ES 6 (2015), what would one have to do?

**Comment by nchanged** [comment]
Hm, what do you mean?

**Comment by devmondo** [comment]
@newtack Fuse compile to `commonjs` `es5` format

**Comment by newtack** [comment]
What I mean is that the final code that is generated is es6/es2015 format, e.g. class syntax, lambda operators, import/export, etc. rather than transpiled back to older es5 code.

**Comment by nchanged** [comment]
Right, but you have Babel and/or TypeScript for that, now do you not?

**Comment by newtack** [comment]
Per the readme:
"It currently works only with TypeScript, and uses it to transpile from es6 to es5."

My question is: Is this hardwired in Fusebox or can I have es6 target? If so, how would the example need to change to have the final output be es6?

**Comment by nchanged** [comment]
@newtack I am afraid it's not possible atm.
